docs/specs/src/hwp_1_intro.md

```diff
@@ -0,0 +1,212 @@
+Blockchain based systems with the approach taken by Holochain as well as
+providing formal reasoning for the benefits of Holochain's agent centric
+approach.
+
+In this second version, we describe Holochain on its own terms by
+providing the highest level view of our approach and our starting
+axioms. At the next level of detail we provide an informal system
+description using the metaphor of games. Then we dive into deeper detail
+with a formal system description including the specific context and
+assumptions we come from, the integrity guarantees Holochain offers, and
+a formal state model and discussion of security and safety concerns.
+Finally at the most detailed level we provide an implementation
+specification in Appendix A.
+
+Axioms
+------
+
+We begin by stating our axioms[^1] regarding the nature of coordination:
+
+1.  **Coordination arises from agents** starting from the same
+  ground-rules and acting as soon as each party in any interaction
+  can confirm that each other\'s actions confirm that actions or
+  interactions conform to those ground rules. (Thus, in our frame,
+  coordination looks like alignment heading in the same direction,
+  rather than agreement proceeding in lockstep.)
+
+2.  **Coordination is grammatic**, in that it comes from embodying a
+  geometry that removes uncertainty, and embodying an ability to
+  compose different coordinative subsystems that have different
+  ground-rules.
+
+Axiom 1 arises from the insight that we cannot fight against the
+physical reality of different experiences by different nodes in the
+context of networked interaction nor fight against the real-world
+impossibility of determining global temporal order because of the
+non-existence of simultaneity. Instead, any coordination system must
+align its ontology with the truth that **global state actually does not
+exist**. Thus we start with what does actually exist: local temporal
+state. This local state can be shared with, and validated by, others in
+conformity with pre-defined ground-rules. In so doing we can still
+achieve difficult and complex coordination safely (including in the
+context of problems on the scale of global monetary transactions)
+without the costs and bottlenecks that arise from starting from the
+ontology of a single shared global state. Holochain is an implementation
+of a system using this alternate frame.
+
+Axiom 2 arises from the insight that systems for successful large scale
+coordination demand the property of anti-fragility, that is, they must
+perform better under perturbation[^2]. Coordination happens in the
+context of fundamentally dynamic environments in which the coordinating
+elements are changed by the fact of their coordination. Coordination is
+a co-evolutionary context. We claim by this axiom that what meets the
+challenge of anti-fragility in such contexts is composable sub-systems,
+in which the composition comes out of a grammatics that embodies the
```

Comment:
```suggestion
in which the composition comes out of a grammar that embodies the
```

Reply:
I see you resolved this, which I assume means you prefer "grammatics", but I wonder what it means, since it's not a word I see used anywhere else. Is it like "application of grammars"?

Reply:
Yes, I'm trying to introduce this notion of "grammatic capacites"  I'm using that specifically because "grammar" by itself makes people think of language, and "grammatic" is my way of generalizing about that.  I will add a footnote.

Reply:
```
FOOTNOTE for "grammatic"
We use the term grammatic as any a way to indicate the general pattern of what grammar and individual grammars are an instance of.  Where grammar is often understood to be limited to language, grammatics points to the pattern of creating templates with classes of items that can fill those templates.  This pattern can be used for creating "grammars" of social interaction,  "grammars" of physical structures (we would call Alexander's "Pattern Language" for architecture a particular instance of grammatics) and so on. 
```